### PR TITLE
update

### PR DESCRIPTION
**Pull request by williamvietnam**

### CODE REVIEW

File: android-mvvm-kotlin/app/src/main/java/com/base/mvvm/core/BaseApplication.kt [hunk 1]
```diff
@@ -10,6 +11,9 @@ class BaseApplication @Inject constructor() : Application() {
 
     override fun onCreate() {
         super.onCreate()
-        Timber.d("BaseApplication()")
+        if (BuildConfig.DEBUG) {
```
Comment (williamvietnam) [inline android-mvvm-kotlin/app/src/main/java/com/base/mvvm/core/BaseApplication.kt:14]: check app is debug or release, release version will delete logger

File: android-mvvm-kotlin/app/src/main/java/com/base/mvvm/samples/ui/login/LoginActivity.kt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.base.mvvm.samples.ui.login
+
```
Comment (williamvietnam) [inline android-mvvm-kotlin/app/src/main/java/com/base/mvvm/samples/ui/login/LoginActivity.kt:2]: create login screen, check login isLoginFirst

File: android-mvvm-kotlin/app/src/main/res/layout/activity_main_sample.xml [hunk 3]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools">
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context=".samples.ui.container.MainSampleActivity">
```
Comment (williamvietnam) [inline android-mvvm-kotlin/app/src/main/res/layout/activity_main_sample.xml:5]: define context to click come back from layout to kotlin code file

File: android-mvvm-kotlin/app/src/main/res/layout/fragment_author_sample.xml [hunk 4]
```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android">
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context="com.base.mvvm.samples.ui.author.AuthorSampleFragment">
```
Comment (williamvietnam) [inline android-mvvm-kotlin/app/src/main/res/layout/fragment_author_sample.xml:4]: define context to click come back from layout to kotlin code file

File: android-mvvm-kotlin/app/src/main/res/layout/fragment_home_sample.xml [hunk 5]
```diff
@@ -19,8 +20,7 @@
             android:id="@+id/root"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:orientation="vertical"
-            tools:context=".samples.ui.home.HomeSampleFragment">
```
Comment (williamvietnam) [inline android-mvvm-kotlin/app/src/main/res/layout/fragment_home_sample.xml:23]: define here not helpful

**Comment by giangnbvietnam** [comment]
nothing to do

File: android-mvvm-kotlin/app/src/main/res/layout/activity_main_sample.xml [hunk 6]
```diff
@@ -14,14 +15,16 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:background="@color/white"
-        android:orientation="vertical"
-        tools:context=".samples.ui.container.MainSampleActivity">
```
Comment (giangnbvietnam) [inline android-mvvm-kotlin/app/src/main/res/layout/activity_main_sample.xml:18]: define context to click come back from layout to kotlin code file

Comment (williamvietnam) [reply android-mvvm-kotlin/app/src/main/res/layout/activity_main_sample.xml:18]: yes

**Pull request merged**